### DISCUSSION TITLE
Indexing XML file with namespace

### DISCUSSION THREAD

**Discussion by paroar** [dbmdz/solr-ocrhighlighting#178] [Q&A] [answered]
Hi, 

is indexing(lazy load) XML files with namespaces supported by the plugin/Solr?

Actually, when I try to index them I get a Solr error indicating XML malformation, but if I get rid of the namespaces then it works fine. Is there a way to index them without the need to change the original file?

Any help or pointer would be appreciated! Thanks.

**Comment by jbaiter** [comment, marked as answer]
Could you open an issue with a sample of your namespaced XML (I'm assuming ALTO?) and the trace of the Solr error?
This could be a bug, namespaced XML should work just fine, at least starting with version 0.6 when we switched to a proper XML parser. If it's not, it's a bug and should be fixed :-)
